megalinter/tests/test_megalinter/plugins_test.py

```diff
@@ -23,12 +23,13 @@ def test_load_plugin_success(self):
         try:
             local_repo = Repo(search_parent_directories=True)
             local_branch = local_repo.active_branch.name
+
         except:  # noqa: E722
             local_branch = "master"
         mega_linter, output = utilstest.call_mega_linter(
             {
                 "PLUGINS": f"https://raw.githubusercontent.com/{ML_REPO}/"
-                + local_branch
+                + "main"
```

Comment:
I hard-coded main in this instance as {ML_REPO} wasn't successfully pointing to my fork

Reply:
Sounds good. I wrote a little blurb as to why the change was made with a note to maybe change in the future.